src/classes/CRLP_RollupUI_SVC.cls

```diff
@@ -513,6 +513,8 @@ public with sharing class CRLP_RollupUI_SVC {
     */
     public static Map<String, String> getAppLabels() {
 
+        PageReference settingsPage = new PageReference('/apex/STG_SettingsManager');
```

Comment:
Another way to write this is as  `PageReference settingsPage = Page.STG_SettingsManager;` which is probably better long term since the page reference is bound as opposed to generated dynamically.

Reply:
@force2b ready for review again!